core/domain/stats_domain_test.py

```diff
@@ -482,6 +482,20 @@ def test_from_dict(self):
                 'schema_version': 1
             })
 
+    def test_from_backend_dict(self):
+        """Test the from_backend_dict() method."""
+        # Test that a playthrough dict without 'exp_id' key raises exception.
+        playthrough_dict = {
+            'exp_version': 1,
+            'issue_type': 'EarlyQuit',
+            'issue_customization_args': {},
+            'actions': []
+        }
+        with self.assertRaisesRegexp(
+            utils.ValidationError,
+            'exp_id not in playthrough data dict.'):
```

Comment:
Can this be indented one level in?

Reply:
@brianrodri Nope. It throws a lint error.